v0/destinations/adobe_analytics/transform.js

```diff
@@ -288,10 +288,14 @@ const processTrackEvent = (
         productMerchEventToAdobeEvent[event.toLowerCase()] &&
         productMerchProperties
       ) {
+        adobeProdEvent = adobeProdEvent.toString().split(",");
```

Comment:
Can we avoid reusing the same variable to represent the event string and its string split array?

Reply:
addressed